tools/releaseBuild/azureDevOps/releaseBuild.yml

```diff
@@ -16,6 +16,7 @@ resources:
     type: github
     endpoint: ComplianceGHRepo
     name: PowerShell/compliance
+    ref: master
```

Comment:
I believe this is the default and it needs to be in `ref/heads/master` format
https://docs.microsoft.com/en-us/azure/devops/pipelines/process/resources?view=azure-devops&tabs=schema#resources-repositories

Reply:
branch name works and I'm contantly having to add and remove this.  I'd like to keep it for easier testing, and returning it back to it's default.

Reply:
See commit:  [`cc463a6` (#14060)](https://github.com/PowerShell/PowerShell/pull/14060/commits/cc463a61e14c88a0b837a548d38748f392873d15)